### PR TITLE
Update configurations up to 3.2 and 4.4

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: rel-eng/releasers.conf [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 
 [koji-foreman-client]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.2-rhel7 foreman-client-3.2-el8 foreman-client-3.2-fedora29
 
 [koji-foreman-client-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.2-rhel7 foreman-client-3.2-el8 foreman-client-3.2-fedora29
```
Comment (evgeni) [inline rel-eng/releasers.conf:31]: we don't release client for fedora29, and @pcreech wanted to clean up the config in develop for a while now

File: rel-eng/releasers.conf [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 builder = tito.builder.FetchBuilder
 
 # Katello Releasers
 [koji-katello]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel7 katello-nightly-el8
+autobuild_tags = katello-3.2-rhel7 katello-3.2-el8
```
Comment (evgeni) [inline rel-eng/releasers.conf:37]: katello is 4.4 ;)

File: rel-eng/releasers.conf [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
 builder.rpmbuild_options = --define "foremandist .fm3_2"
 
 [koji-katello-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = katello-nightly-rhel7 katello-nightly-el8
+autobuild_tags = katello-3.2-rhel7 katello-3.2-el8
```
Comment (evgeni) [inline rel-eng/releasers.conf:42]: same

File: rel-eng/tito.props [hunk 4]
```diff
@@ -977,25 +977,25 @@ whitelist =
...
+# don't whitelist foreman-release here, it's already in foreman-3.2-nonscl-rhel7
 whitelist =
   katello-host-tools
   python-psutil
   rubygem-foreman_scap_client
   tracer
   yggdrasil
 
-[foreman-client-nightly-fedora29]
+[foreman-client-3.2-fedora29]
```
Comment (evgeni) [inline rel-eng/tito.props:990]: drop it

File: rel-eng/tito.props [hunk 5]
```diff
@@ -977,25 +977,25 @@ whitelist =
...
 disttag = .fc29
 whitelist =
   foreman-release
   katello-host-tools
   rubygem-foreman_scap_client
   tracer
 
-[katello-nightly-rhel7]
+[katello-3.2-rhel7]
```
Comment (evgeni) [inline rel-eng/tito.props:998]: katello 4.4

File: rel-eng/tito.props [hunk 6]
```diff
@@ -1041,7 +1041,7 @@ whitelist = katello
   rubygem-typhoeus
   rubygem-zest
 
-[katello-nightly-el8]
+[katello-3.2-el8]
```
Comment (evgeni) [inline rel-eng/tito.props:1044]: 4.4

**Comment by zjhuntin** [comment]
@evgeni thanks, fixed!

**Review by pcreech — APPROVED** [review]

**Pull request merged**